tests/integration/test_wazuh_db/data/global_messages.yaml

```diff
@@ -219,12 +219,12 @@
     stage: "global sync-agent-info-set success"
```

Comment:
Considering that in the cluster synchronization all the agent data is sent from the worker to the master. Shouldn't we add in the `sync-agent-info-set` a value for `connection_status`?

Reply:
That is logic, but currently, the **wdb_global_sync_agent_info_set()** method would ignore the field **connection_status**.